### DISCUSSION TITLE
Can inav4.1 add firmware for Rush_blade_F7_HD(STM32F7X2)?

### DISCUSSION THREAD

**Discussion by elfox-14482** [iNavFlight/inav#8037] [Q&A] [answered]
one of my quad running this flight controller. Really wanna to use Inav. can anyone helf with the hex?

**Comment by DzikuVx** [comment, marked as answer]
[inav_4.1.0_RUSH_BLADE_F7.zip](https://github.com/iNavFlight/inav/files/8695003/inav_4.1.0_RUSH_BLADE_F7.zip)

**Reply by elfox-14482** [reply]
Thx a lot,man~~

**Reply by elfox-14482** [reply]
hello, DzikuVx~~ Is there Included barometer function in the firmware that you gave me? coz when I configure the inav, the barometer works normally only when MSP is selected. Is this the right setting?

**Reply by elfox-14482** [reply]
when i check the BetaFlight setting, they had "set baro_bustype = I2C", "set baro_i2c_device = 1" in the setting. is this setting need to write into the firmware?  coz i cant find this in CLI of Inav

**Reply by elfox-14482** [reply]
and also i check my flight controller, it should use BMP280，but now is not working when i select the BMP280 protocol。

**Reply by DzikuVx** [reply]
Are you using INAV Configurator 4.1? I have a feeling that you don't

**Reply by elfox-14482** [reply]
yes, inav 4.1

**Reply by elfox-14482** [reply]
i try "AUTO protocol"  for the baro, when it save and comeback to the setting page , it show None. looks cant read the baro correctly

**Reply by elfox-14482** [reply]
hello DzikuVx, just want to confirm that when i flash the firmware of inav , do i need to load a config doc first? just like BF?
